src/auth/index.ts

```diff
@@ -367,14 +367,20 @@ export namespace auth {
    * Interface representing a phone specific user enrolled second factor
    * for an `UpdateRequest`.
    */
-  export interface UpdatePhoneMultiFactorInfoRequest extends UpdateMultiFactorInfoRequest {
+  export interface UpdatePhoneMultiFactorInfoRequest extends BaseUpdateMultiFactorInfoRequest {
 
     /**
      * The phone number associated with a phone second factor.
      */
     phoneNumber: string;
   }
 
+  /**
+   * Type representing the properties of a user enrolled second factor
```

Comment:
This would be clearer as "user-enrolled" 

There are 7 other instances in the file; if that's a PITA, I can do it in a follow-on.

Reply:
Fixed